sciml/models/gefno/gfno.py

```diff
@@ -8,7 +8,7 @@
 import torch.nn as nn
 import math
 
-from neuralop.models.padding import DomainPadding
```

Comment:
I think this needs to be undone. I think this was changed sometime between neuralop versions 0.2.0 and now... Since our checkpoints are stuck on 0.2.0, we're sort of stuck for the moment.
(I updated the conda file to keep it on 0.2.0)